react/src/components/molecules/EmergencyAlert/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import is from 'is';
+import './style.css';
+
+const EmergencyAlert = (props) => {
+  const { message, timeStamp, link } = props;
+  const theme = props.theme.length === 0 ? 'c-highlight' : props.theme;
+  const linkArgs = {
+    theme,
+    linkClasses: 'ma__content-link',
+    TextElement: 'span'
+  };
+  return(
+    <p className={`ma__emergency-alert ma__emergency-alert--${theme}`}>
+      <span className="ma__emergency-alert__message">
+        {message}
+        {(timeStamp) && (<span className="ma__emergency-alert__time-stamp">&nbsp;{timeStamp}</span>)}
+      </span>
+      {is.fn(link) && (
+        <span className="ma__emergency-alert__link">
+          &nbsp;{link({ ...linkArgs })}
+        </span>
+      )}
+    </p>
+  );
+};
+
+EmergencyAlert.propTypes = {
+  /** A string that controls different color themes for the component. */
+  theme: PropTypes.oneOf(['', 'c-highlight', 'c-primary-alt', 'c-primary', 'c-gray', 'c-error']),
```

Comment:
since there's a default prop for theme, i think we don't need an empty default
```suggestion
  theme: PropTypes.oneOf(['c-highlight', 'c-primary-alt', 'c-primary', 'c-gray', 'c-error']),
```